tools/msvs/find_python.cmd

```diff
@@ -74,27 +48,6 @@ for /f "skip=2 tokens=1* delims=)" %%a in ('reg query "%1\InstallPath" /ve /t RE
 exit /b 1
 
 
-:: Python 2: Check if %p% holds a path to a real python2 executable
-:validate-v2
-IF NOT EXIST "%p%\python.exe" EXIT /B 1
-:: Check if %p% is python2
-"%p%\python.exe" -V 2>&1 | findstr /R "^Python.2.*" > NUL
-EXIT /B %ERRORLEVEL%
-
-
-:: Python 2:
-:found-python2
-echo Python 2 found in %p%\python.exe
-set pyver=2
-goto :done
-
-:found-python
```

Comment:
I think we need to keep this block and remove the warning. But I don't know how to set `pyver` to the right value
/cc @joaocgreis